nodejs/aws-serverless/examples/cloudwatch/index.ts

```diff
@@ -13,15 +13,12 @@
 // limitations under the License.
 
 import * as aws from "@pulumi/aws";
-import * as serverless from "@pulumi/aws-serverless";
-import * as pulumi from "@pulumi/pulumi";
-import { Output } from "@pulumi/pulumi";
+import * as cloudwatch from "@pulumi/aws-serverless/cloudwatch";
+import * as sns from "@pulumi/aws-serverless/sns";
 
 const topic = new aws.sns.Topic("topic", { });
 
-serverless.topic.subscribe("process-topic", topic, async (event) => {
-    const awssdk = await import("aws-sdk");
-
+sns.onEvent("process-topic", topic, async (event) => {
```

Comment:
Is "Event" the right word for a new item delivered to a topic?  Is this the word that AWS docs use?

Reply:
AWS both talks about "message" and "event".  "sns onMessage".